my-app/src/store/meta.ts

```diff
@@ -0,0 +1,6 @@
+export enum Meta {
+  initial = "initial", // Процесс не начат
```

Comment:
можно оставлять комменты над переменными/функциями/полями енума и т.д. в формате [JSDoc](https://jsdoc.app/), тогда IDE-шка сможет давать подсказки при наведении кусора на значение
```suggestion
  /**
   * Процесс не начат
   */
  initial = "initial",
```
<img width="350" alt="Снимок экрана 2023-03-09 в 00 31 41" src="https://user-images.githubusercontent.com/56839604/223787255-5b00615d-8599-4830-aa50-24a59d792a54.png">